bootstrap/dev/venv.sh

```diff
@@ -11,3 +11,5 @@ export VENV_ARGS="--python python2"
   -e letsencrypt-nginx \
   -e letshelp-letsencrypt \
   -e letsencrypt-compatibility-test
+
+find . -iname '*.pyc' -exec rm '{}' ';'
```

Comment:
What if someone (accidentally) run `/path/to/bootstrap/dev/venv.sh` from a directory which is not a part of LE but contains `*.pyc` files?